Dockerfile.dapper

```diff
@@ -116,6 +117,7 @@ ENV BUILD_DOCKER_URL=BUILD_DOCKER_URL_${ARCH} \
     SYSTEM_DOCKER_URL_amd64=${SYSTEM_DOCKER_URL_amd64} \
     SYSTEM_DOCKER_URL_arm64=${SYSTEM_DOCKER_URL_arm64} \
     VMWARE_AUTOFORMAT=${VMWARE_AUTOFORMAT} \
+    VMWARE_APPEND=${VMWARE_APPEND} \
```

Comment:
Same here.

Add `VMWARE_APPEND` to `ENV DAPPER_ENV ... VMWARE_APPEND`

Reply:
Already modified